### PR TITLE
fix: adding in ingress and values for multiple hosts

### PR DESCRIPTION
**Pull request by jmedberydsc**
- Allowing for multiple values to be passed into annotations
- Allowing for multiple values to be passed for hosts
- default settings for values

### CODE REVIEW

File: .helm/charts/acyl/templates/ingress.yaml [hunk 1]
```diff
@@ -7,18 +8,19 @@ metadata:
...
-    - host: {{ .Values.env_name }}-{{ .Values.ingress.argo_host }}
-    {{- else }}
-    - host: {{ .Values.ingress.argo_host }}
-    {{- end }}
+    {{- $serviceName := .Values.service.name -}}
+    {{- range $host := .Values.ingress.hosts }}
+    - host: {{ $host }}
       http:
-        paths:
+      	paths:
```
Comment (cleanshavenalex) [inline .helm/charts/acyl/templates/ingress.yaml:20]: is this too indented?

**Review by cleanshavenalex — COMMENTED** [review]
left a comment

**Review by cleanshavenalex — APPROVED** [review]
+1

File: .helm/charts/acyl/templates/ingress.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{{- $servicePort := .Values.service.internalPort -}}
```
Comment (mikeykhalil) [inline .helm/charts/acyl/templates/ingress.yaml:1]: `servicePort` should be defined right below `serviceName`. This reduces the the scope of the variable and also reduces the distance between where it's defined and where it is used.

**Review by mikeykhalil — APPROVED** [review]
Left one small comment, but looks good to me!

**Review by mikeykhalil — APPROVED** [review]

**Pull request merged**